### PR TITLE
Closes #71: Fix threshold bands for below zero values

### LINKED ISSUE #71: Threshold bars are weird if y axis includes negative values
When the feature was written we only had mock data whose values were all > 0. 

If the y scale includes 0 and below, the threshold feature freaks out (top: bad, bottom: good):
screen shot 2016-09-20 at 12 51 26 pm

General clean up of bar heights necessary all around, too.

### PR DESCRIPTION
**Pull request by sharph**
## Overview

This fixes the broken visualization of data above or below thresholds as described in issue #71. It also specifies a fixed height to match the charting area.

### Demo

bands

## Testing Instructions

* Choose a city with indicators with negative numbers. Anchorage, AK, for example.
* Pick an indicator with negative numbers (Yearly Minimum Low Temperature)
* Make sure threshold display does not look broken

Closes #71

### CODE REVIEW

File: src/app/charts/line-graph.component.ts [hunk 1]
```diff
@@ -270,4 +270,20 @@ export class LineGraphComponent {
           .attr('y', d => this.yScale(d.value))
           .attr('height', d => this.height - this.yScale(d.value));
```
Comment (fungjj92) [inline src/app/charts/line-graph.component.ts]: drawBarGraph isn't used anymore so can be deleted

File: src/app/charts/line-graph.component.ts [hunk 2]
```diff
@@ -216,34 +216,34 @@ export class LineGraphComponent {
...
+                              .filter((bar, index) => this.minVal > this.yData[index])
+                              .value();
 
                 // Add bar graph
-                this.drawBarGraph(minBars, 'min-bar');
+                this.drawBands(minBars, 'min-bar');
             }
 
-            if (this.max && this.maxVal) {
+            if (this.max) {
```
Comment (fungjj92) [inline src/app/charts/line-graph.component.ts]: For this, we'd need to keep `&& this.maxVal` and `&& this.minVal` or the dotted line shows up and in sometimes undesirable places (e.g. below graph)

Comment (sharph) [inline src/app/charts/line-graph.component.ts]: Having `&& this.maxVal` had the effect of hiding the bars when `maxVal` was set to zero, as it is falsey. If I leave `&& this.maxVal` in, the line still shows in inappropriate places when it is negative.

This should really be comparing `maxVal` and `minVal` against the `y` range of the chart. I'll look into adding this.

Comment (sharph) [inline src/app/charts/line-graph.component.ts]: Addressed in https://github.com/azavea/climate-change-lab/pull/92/commits/1f026f58154df5c52a9000144b07bea60ba83c1f

**Comment by sharph** [comment]
Ready for another look!

File: src/app/charts/line-graph.component.ts (hunk 2 above)
Comment (fungjj92) [inline src/app/charts/line-graph.component.ts]: Nice

**Comment by fungjj92** [comment]
+1

**Pull request merged**